internals/secrethub/run.go

```diff
@@ -422,6 +422,48 @@ func ReadEnvFile(filepath string, varReader tpl.VariableReader, parser tpl.Parse
 	}, nil
 }
 
+// OsEnv is an environment with secrets configured with the
+// secrethub:// syntax in the os environment variables.
+type OsEnv struct {
```

Comment:
Any ideas for a better name?

Reply:
This is a though one for naming. Maybe something with reference? As environment variable contains a secret reference. e.g. `referenceEnv`.

Whatever the name, please make it private. Then we can always change the name later. When making this functionality public to other packages (e.g. to be used from end-user Golang applications), we should move it to its own package.

Reply:
What about `SecretReferenceEnv`?

Reply:
`referenceEnv` might be better.

Reply:
`referenceEnv` might be better. And I agree on it being unexported.

Reply:
`referenceEnv` sounds good :+1: 
Made it private.